packages/devtools/src/config/env.ts

```diff
@@ -1,44 +1,47 @@
-"use strict";
-import fs from "fs";
-import path from "path";
+'use strict';
+import fs from 'fs';
+import path from 'path';
 
 export function configureEnvVariables(monorepo = true) {
   const NODE_ENV = process.env.NODE_ENV;
   if (!NODE_ENV) {
-    throw new Error("The NODE_ENV environment variable is required but was not specified.");
+    throw new Error('The NODE_ENV environment variable is required but was not specified.');
   }
 
   // https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use
   let dotenvFiles = [
     `.env.${NODE_ENV}.local`,
     // Don't include `.env.local` for `test` environment
-    // since normally you expect tests to produce the same
-    // results for everyone
-    NODE_ENV !== "test" && `.env.local`,
+    // Since normally you expect tests to produce the same
+    // Results for everyone
+    NODE_ENV !== 'test' && `.env.local`,
     `.env.${NODE_ENV}`,
-    ".env"
+    '.env'
   ].filter(Boolean);
 
   if (monorepo) {
     const monorepoDotenvFiles = dotenvFiles.slice(0);
     dotenvFiles.forEach((dotenvFile: string) => {
-      monorepoDotenvFiles.push(path.join("../..", dotenvFile));
+      monorepoDotenvFiles.push(path.join('../..', dotenvFile));
     });
     dotenvFiles = monorepoDotenvFiles;
   }
 
   // Load environment variables from .env* files. Suppress warnings using silent
-  // if this file is missing. dotenv will never modify any environment variables
-  // that have already been set.  Variable expansion is supported in .env files.
+  // If this file is missing. dotenv will never modify any environment variables
```

Comment:
It doesn't make sense for this to be capitalized - it's the middle of a sentence.